components/Footer.js

```diff
@@ -3,38 +3,69 @@ import Newsletter from './Newsletter'
 import Link from 'next/link'
 import api from '../utils/api'
 import { trackEvent } from './Analytics'
-import { PLATFORM_NAME, LOGO_FOOTER, TEAM_EMAIL, INSTAGRAM_URL, FACEBOOK_URL, TWITTER_URL, NEWSLETTER, DISCORD_URL } from '../config';
+import { PLATFORM_NAME, LOGO_FOOTER, LOGO_HEADER, TEAM_EMAIL, INSTAGRAM_URL, FACEBOOK_URL, TWITTER_URL, NEWSLETTER, DISCORD_URL } from '../config';
 
 const footer = () => (
-  <div className="footer-wrapper no-print bg-primary">
-    { NEWSLETTER && <Newsletter placement="Footer" /> }
-    <footer className="main-content">
-      <div className="flex flex-row items-center justify-between">
-        <div className="flex flex-row items-center justify-start">
+  <div className="bg-slate-50  border-t border-t-neutral-700">
```

Comment:
bg color not required. Also - can you create a new color in tailwind.config, in the list of colors add a `line` or `border` color that you can then re-use for all these lines like `border-t-line` instead of having to put a custom color each time.

Reply:
Created 'line' color